### ISSUE TITLE
Add metadata and screenshots to the repository

### ISSUE THREAD

**Issue by locness3** [Scratch-Client-4/itchy-ionic#64]
Speaking of contributions, we should make [the application] screenshots and metadata part of this repository, following [the format](https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/#fastlane-structure) used by the [fastlane supply tool](https://github.com/fastlane/fastlane/blob/2.28.7/supply/README.md#images-and-screenshots) (which we could use as well), so they can be picked up by F-Droid and potentially automatically uploaded to Google Play via that fastlane thing, as well as to make it possible to collaboratively work on them and accept contributions.

_Originally posted by @locness3 in https://github.com/Scratch-Client-4/itchy-ionic/issues/63#issuecomment-1102860146_

**Comment by micahlt** [comment]
Screenshots are already here, as well as a source file in the .afdesign format: https://github.com/Scratch-Client-4/itchy-ionic/tree/master/resources/screenshots

**Comment by locness3** [comment]
Oh, thanks for providing a source file. It would be helpful to have it in an open format though.

**Comment by micahlt** [comment]
I agree - when I get around to it, I might be able to provide an SVG.  However, at the moment it uses program-specific features that can't be converted to SVG without rasterization.
